org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/utils/FileExclusionsChecker.java

```diff
@@ -71,13 +72,17 @@ public FileExclusionsChecker(ISonarLintProject project) {
       .resolveBinding(project)
       .ifPresent(binding -> {
         TestFileClassifier testFileClassifier = TestFileClassifier.get();
-        List<ISonarLintFile> excludedByServerSideExclusions = binding.getEngineFacade().getServerFileExclusions(binding.getProjectBinding(), notExcluded,
-          testFileClassifier::isTest);
-        notExcluded.removeAll(excludedByServerSideExclusions);
-        excludedByServerSideExclusions.forEach(file -> {
-          notExcluded.remove(file);
-          logIfNeeded(file, log, "server side");
-        });
+        try {
+          List<ISonarLintFile> excludedByServerSideExclusions = binding.getEngineFacade().getServerFileExclusions(binding.getProjectBinding(), notExcluded,
+            testFileClassifier::isTest);
+          notExcluded.removeAll(excludedByServerSideExclusions);
+          excludedByServerSideExclusions.forEach(file -> {
+            notExcluded.remove(file);
+            logIfNeeded(file, log, "server side");
+          });
+        } catch (SonarLintException e) {
+          SonarLintLogger.get().error("Unable to read server side exclusions. Check your binding.", e);
```

Comment:
This change is not strictly speaking related to the ticket, but I noticed that the error was not properly handled during my tests.